builder/azure/arm/config.go

```diff
@@ -382,6 +382,9 @@ type Config struct {
 	// `virtual_network_name` is not allowed.
 	AllowedInboundIpAddresses []string `mapstructure:"allowed_inbound_ip_addresses"`
 
+	//Spcify storage to store Boot Diagnostics
```

Comment:
Is there any additional information that our uses should be aware of when using the option? For instance, I don't see a clean up process for the boot diagnostics is that something that users will be responsible for cleaning up once the build has terminated?

Reply:
Probably also fix the typo here, since the docs are generated from it.

Boot diagnostics means that a serial console log and a screenshot are streamed to the storage account. Generally, these are not automatically cleaned up in Azure when a VM is deleted, so this would be for the customer to clean up if that is what they want.

Reply:
@nywilken @paulmey I hope the new changes are sufficient